### ISSUE TITLE
Theme.json: Add granular control to appearance tools

### ISSUE THREAD

**Issue by scruffian** [WordPress/gutenberg#47925]
## What problem does this address?
An idea that arose in https://github.com/WordPress/gutenberg/pull/47675 is that we could give theme.json more granular control of the appearance tools that are enabled in a theme.

## What is your proposed solution?
We could use a syntax like this:
```
    "settings": {
        "appearanceTools": {
            'link-color' => true,
            'border'  => true,
        },    
    },
 ```

**Comment by bph** [comment]
@scruffian Looking at [the documentation](https://developer.wordpress.org/block-editor/reference-guides/theme-json-reference/theme-json-living/#appearancetools) , this seems to be completed. If you concur, we can close this issue..

**Comment by scruffian** [comment]
I don't think so. At the moment appearanceTools just turns on all of those settings.

**Comment by youknowriad** [comment]
You can set "appearanceTools" to false and enable the settings granularity.
I also verified on current WordPress that individual tools enabled by appearanceTools can be opted out granularly — an explicit false (e.g. settings.border.radius: false) overrides the shorthand while the rest stays on. That's the granular control this asked for. 

Closing this, let me know if I missed anything though.
